tests/xharness/Harness.cs

```diff
@@ -704,6 +704,27 @@ public void LogWrench (string message)
 				return string.IsNullOrEmpty (groupApps) || groupApps == "grouped";
 			}
 		}
+		
+		public bool SkipTvOSDeviceVariations {
+			get {
+				var skipTv = Environment.GetEnvironmentVariable ("SKIP_TVOS_VARIATIONS");
+				return !string.IsNullOrEmpty (skipTv) && skipTv == "true";
```

Comment:
Why the `IsNullOrEmpty` check? if `skipTv` is null, the test will be false, as you'd like.

Reply:
@stephen-hawley I followed the pattern used in the rest of the properties. In my case, just following the style already present.. don't know if that is a good reason enough.